pulsar-broker/src/main/java/com/yahoo/pulsar/broker/loadbalance/impl/SimpleLoadManagerImpl.java

```diff
@@ -177,7 +177,7 @@ public SimpleLoadManagerImpl(PulsarService pulsar) {
         placementStrategy = new WRRPlacementStrategy();
         lastLoadReport = new LoadReport(pulsar.getWebServiceAddress(), pulsar.getWebServiceAddressTls(),
                 pulsar.getBrokerServiceUrl(), pulsar.getBrokerServiceUrlTls());
-        brokerHostUsage = new BrokerHostUsage(pulsar);
+        brokerHostUsage = new LinuxBrokerHostUsageImpl(pulsar);
```

Comment:
I mean, we don't need precise load information for MacOS, since it's not meant to be used for prod.. just not throwing exception or printing error logs, at least for now.